sdk/servicebus/azure-servicebus/azure/servicebus/management/_management_client.py

```diff
@@ -0,0 +1,317 @@
+            else BearerTokenCredentialPolicy(self._credential, JWT_TOKEN_SCOPE)
+        if policies is None:  # [] is a valid policy list
+            policies = [
+                RequestIdPolicy(**kwargs),
+                self._config.headers_policy,
+                self._config.user_agent_policy,
+                self._config.proxy_policy,
+                ContentDecodePolicy(**kwargs),
+                self._config.redirect_policy,
+                self._config.retry_policy,
+                credential_policy,
+                self._config.logging_policy,
+                DistributedTracingPolicy(**kwargs),
+                HttpLoggingPolicy(**kwargs),
+            ]
+        if not transport:
+            transport = RequestsTransport(**kwargs)
+        return Pipeline(transport, policies)
+
+    @classmethod
+    def from_connection_string(cls, connection_string, **kwargs):
+        # type: (str, Any) -> ServiceBusManagementClient
+        """Create a client from connection string
+
+        :param str connection_string: The connection string of the Service Bus Namespace
+        """
+        endpoint, shared_access_key_name, shared_access_key, _ = parse_conn_str(connection_string)
+        if "//" in endpoint:
+            endpoint = endpoint[endpoint.index("//")+2:]
+        return cls(endpoint, ServiceBusSharedKeyCredential(shared_access_key_name, shared_access_key), **kwargs)
+
+    def _get_queue_object(self, queue_name, **kwargs):
+        # type: (str, Any) -> InternalQueueDescription
+
+        if not queue_name:
+            raise ValueError("queue_name must be a non-empty str")
+
+        with _handle_response_error():
+            et = cast(
+                ElementTree,
+                self._impl.queue.get(queue_name, enrich=False, api_version=constants.API_VERSION, **kwargs)
+            )
+        return _convert_xml_to_object(queue_name, et)
+
+    def _list_queues(self, **kwargs):
+        # type: (Any) -> List[Tuple[str, InternalQueueDescription]]
+
+        skip = kwargs.pop("skip", 0)
+        max_count = kwargs.pop("max_count", 100)
+        with _handle_response_error():
+            et = cast(
+                ElementTree,
+                self._impl.list_entities(
+                    entity_type=constants.ENTITY_TYPE_QUEUES, skip=skip, top=max_count,
+                    api_version=constants.API_VERSION, **kwargs
+                )
+            )
+        entries = et.findall(constants.ENTRY_TAG)
+        queues = []
+        for entry in entries:
+            entity_name = entry.find(constants.TITLE_TAG).text  # type: ignore
+            internal_object = _convert_xml_to_object(
+                entity_name,   # type: ignore
+                cast(Element, entry)
+            )
+            queues.append((entity_name, internal_object))
+        return queues   # type: ignore
+
+    def get_queue(self, queue_name, **kwargs):
+        # type: (str, Any) -> QueueDescription
+        """Get a QueueDescription
```

Comment:
"""
Get the properties of a queue.

:rtype: ~azure.servicebus.QueueDescription
"""
In fact... this method should probably be called "get_queue_properties"....

Reply:
Also please add the `:rtype:` tag to all these public docstrings.

Reply:
Added :rtype: too all public docstrings